extensions/lifecycle/tasks/LifecycleTask.js

```diff
@@ -96,10 +95,10 @@ class LifecycleTask extends BackbeatTask {
         });
 
         this._lifecycleUtils = new LifecycleUtils(
-            supportedLifecycleRules,
+            config.extensions.lifecycle.supportedLifecycleRules,
             this._lifecycleDateTime
         );
-        this._supportedRules = supportedLifecycleRules;
+        this._supportedRules = config.extensions.lifecycle.supportedLifecycleRules;
         this._totalRetries = 0;
     }
```

Comment:
Might be better to pass the supported rules through `lp.getStateVars()`, which is called at the beginning of the constructor. This function [lets you pass values from the bucket procesor](https://github.com/scality/backbeat/blob/0a32e66773027f99b5057bf9159bf1ec1f5fc3c0/extensions/lifecycle/bucketProcessor/LifecycleBucketProcessor.js#L176-L190) that has access to the lifecycle config.

Reply:
done here : https://github.com/scality/backbeat/pull/2548/commits/a2b5caaaa07e99197759d1dbc195f1ff4d56358e#diff-0251eb0ff96b3559c00aac5d7e592f81df2eece3b644f9278307237dd8f6bbbcR189